packages/profiling-node/package.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "@sentry/profiling-node",
+  "version": "1.3.5",
+  "description": "Official Sentry SDK for Node.js Profiling",
+  "repository": "git://github.com/getsentry/sentry-javascript.git",
+  "homepage": "https://github.com/getsentry/sentry-javascript/tree/master/packages/profiling-node",
+  "author": "Sentry",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "types": "lib/types/index.d.ts",
+  "typesVersions": {
+    "<4.9": {
+      "lib/types/index.d.ts": [
+        "lib/types-ts3.8/index.d.ts"
+      ]
+    }
+  },
+  "bin": {
+    "sentry-prune-profiler-binaries": "scripts/prune-profiler-binaries.js"
+  },
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "lib",
+    "bindings",
+    "binding.gyp",
+    "LICENSE",
+    "README.md",
+    "package.json",
+    "scripts/binaries.js",
+    "scripts/check-build.js",
+    "scripts/copy-target.js",
+    "scripts/prune-profiler-binaries.js"
+  ],
+  "scripts": {
+    "install": "node scripts/check-build.js",
+    "clean": "rm -rf build && rm -rf lib",
+    "lint": "yarn lint:eslint && yarn lint:clang",
+    "lint:eslint": "eslint . --format stylish",
+    "lint:clang": "node clang-format.js",
+    "fix": "eslint . --format stylish --fix",
+    "lint:fix": "yarn fix:eslint && yarn fix:clang",
+    "lint:fix:clang": "node clang-format.js --fix",
+    "build": "yarn build:lib && yarn build:bindings:configure && yarn build:bindings",
+    "build:lib": "yarn build:types && rollup -c rollup.npm.config.mjs",
+    "build:transpile": "yarn build:bindings:configure && yarn build:bindings && yarn build:lib",
+    "build:types:downlevel": "yarn downlevel-dts lib/types lib/types-ts3.8 --to ts3.8",
+    "build:types": "tsc -p tsconfig.types.json && yarn build:types:downlevel",
+    "build:types:watch": "tsc -p tsconfig.types.json --watch",
+    "build:bindings:configure": "node-gyp configure",
+    "build:bindings:configure:arm64": "node-gyp configure --arch=arm64 --target_arch=arm64",
+    "build:bindings": "node-gyp build && node scripts/copy-target.js",
+    "build:bindings:arm64": "node-gyp build --arch=arm64 && node scripts/copy-target.js",
+    "build:dev": "yarn clean && yarn build:bindings:configure && yarn build",
+    "build:transpile:watch": "rollup -c rollup.npm.config.mjs --watch",
+    "build:watch": "run-p build:transpile:watch build:types:watch",
+    "build:tarball": "npm pack",
+    "test:watch": "cross-env SENTRY_PROFILER_BINARY_DIR=build jest --watch",
+    "test:bundle": "node test-binaries.esbuild.js",
+    "test": "cross-env SENTRY_PROFILER_BINARY_DIR=lib jest --config jest.config.js"
+  },
+  "dependencies": {
+    "detect-libc": "^2.0.2",
+    "node-abi": "^3.52.0"
+  },
+  "devDependencies": {
+    "@sentry/core": "7.93.0",
+    "@sentry/hub": "7.93.0",
+    "@sentry/node": "7.93.0",
+    "@sentry/types": "7.93.0",
+    "@sentry/utils": "7.93.0",
```

Comment:
Why are the `@sentry` packages declared as `devDependencies`? Shouldn't they be `dependencies` given we're importing from them?

Reply:
The library is bundled before published to minimize impact on cold start perf